test/e2e/e2e.go

```diff
@@ -115,17 +131,17 @@ var _ = Describe("validation", func() {
 				Eventually(func() bool {
 					err = testclient.Client.Get(context.Background(), goclient.ObjectKey{Namespace: metallb.Namespace, Name: metallb.Name}, metallb)
 					return errors.IsNotFound(err)
-				}, 5*time.Minute, 5*time.Second).Should(BeTrue(), "Failed to delete MetalLB custom resource")
+				}, timeout, interval).Should(BeTrue(), "Failed to delete MetalLB custom resource")
```

Comment:
it can take some time until MetalLB custom resource is deleted. i wanted to be on the safe side so i set it to 5, i guess 2 min is also enough time.
@msherif1234 maybe we can set the timeout to 3 min?

Reply:
sure